Sources/Jetworking/Client/Client.swift

```diff
@@ -120,6 +143,19 @@ public final class Client {
         return nil
     }
 
+    @discardableResult
+    public func download(
+        url: URL,
+        progressHandler: DownloadHandler.ProgressHandler,
+        _ completion: @escaping DownloadHandler.CompletionHandler
+    ) -> CancellableRequest? {
+        // TODO: Only HTTP and HTTPS protocols are supported (no custom protocols). --> Error Handling
```

Comment:
To Validate this you could use URLComponents and check its scheme property.

Reply:
Done.